### PR TITLE
Bugfix: Change select logo to input logo

### PR DESCRIPTION
**Pull request by giancorzo**
https://developertown.visualstudio.com/SIL%20International%20Scriptoria/_workitems/edit/29190

- [x] Change organization settings to input
- [x] Change sidebar to use logo

logo

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
should we be previewing the logo somewhere?
also, since orgs now have an image url, I think the org-switcher needs updating.
the orgs in the org switcher currently just have grey boxes

**Comment by giancorzo** [comment]
I did update the org switcher. So only thing to add would be the placeholder to show the image

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/org-switcher/row.tsx [hunk 1]
```diff
@@ -15,12 +15,14 @@ class Row extends React.Component<IProps> {
...
     const { isActive, organization, onClick } = this.props;
     const attributes = attributesFor(organization);
-    const { name } = attributes;
+    const { name, logoUrl } = attributes;
 
     const className = `
           flex-row align-items-center
           ${isActive ? 'active' : ''}`;
 
+    const logo = logoUrl ? <img src={logoUrl} width='32' /> : '\u00A0';
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/org-switcher/row.tsx]: :+1:

Comment (NullVoxPopuli) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/org-switcher/row.tsx]: is this the correct size?

Comment (giancorzo) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/org-switcher/row.tsx]: yes it's

Comment (NullVoxPopuli) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/org-switcher/row.tsx]: screenshot? :)

Comment (giancorzo) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/org-switcher/row.tsx]: selecetd
row

**Pull request merged**